### PR TITLE
Create Dependabot configuration

### PR DESCRIPTION
**Pull request by tw-sec**
## Context

Enable Dependabot configuration for the repository. For more detailed information head over to:
[Enabling Dependabot In GitHub Repositories](https://transferwise.atlassian.net/wiki/spaces/SO/pages/1593803057/Enabling+Dependabot+In+GitHub+Repositories)

### Changes

The commit creates a file `.github/dependabot.yml` containing the base configuration needed to enable Dependabot vulnerability scanning in the repository.

## Checklist
- [ ] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [ ] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      - dependency-name: 'react-dom'
+      - dependency-name: 'react-scripts'
+      - dependency-name: 'typescript'
+      - dependency-name: '@transferwise/*'
+      - dependency-name: 'antd'
+      - dependency-name: '@ant-design/*'
+      - dependency-name: 'axios'
+      - dependency-name: 'jest'
+      - dependency-name: '@testing-library/*'
+      - dependency-name: 'nock'
```
Comment (dozortsev) [inline .github/dependabot.yml:23]:
```suggestion
      - dependency-name: 'currency-flags'
      - dependency-name: 'react'
      - dependency-name: 'react-dom'
      - dependency-name: 'typescript'
      - dependency-name: '@transferwise/*'
      - dependency-name: 'jest'
      - dependency-name: '@testing-library/*'
```

Comment (dozortsev) [reply .github/dependabot.yml:23]: @ivang-tw I removed `nock`, `axios`, `antd`, `@ant-design/*` and `react-scripts` as I think we most likely never use them in this repo, any objections?

Comment (helendurranttw) [reply .github/dependabot.yml:23]: Fine with me, not sure if its a security requirement though - @ivang-tw?

Comment (ivang-tw) [reply .github/dependabot.yml:23]: The suggested config is just a template, feel free to adjust it to your needs adding/removing whatever you need.

Comment (dozortsev) [reply .github/dependabot.yml:23]: ok

**Review by dozortsev — APPROVED** [review]

**Pull request merged**